lib/xdrgen/generators/rust.rb

```diff
@@ -134,6 +134,7 @@ def render_source_comment(out, defn)
 
       def render_struct(out, struct)
         out.puts "#[derive(Clone, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]"
+        out.puts "#[cfg_attr(feature = \"serde\", derive(Serialize, Deserialize), serde(rename_all = \"camelCase\"))]"
```

Comment:
Can you gate the serde code on alloc too?
```suggestion
        out.puts "#[cfg_attr(all(feature = \"serde\", feature = \"alloc\"), derive(Serialize, Deserialize), serde(rename_all = \"camelCase\"))]"
```